### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by luzianz**

### CODE REVIEW

**Comment by horiuchi** [comment]
@luzianz Thank you for your contribution.
But I thought `.gitignore` and `package.json` files is not required.
Please remove these files.

**Comment by luzianz** [comment]
@horiuchi Ah, yes. Sorry about that. This is my first time contributing. It is now corrected.

**Comment by horiuchi** [comment]
@luzianz Thank you for your commit! :smile: 
---
And one more issue, I found that there are no defined APIs in the `node-calendar` github repository.
https://github.com/ArminTamzarian/node-calendar#calendarisleapyear

File: node-calendar/node-calendar.d.ts [hunk 1]
```diff
@@ -0,0 +1,271 @@
+// Type definitions for node-calendar v0.1.4
+// Project: https://www.npmjs.com/package/node-calendar
+// Definitions by: Luzian Zagadinow <https://github.com/luzianz>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+interface IDayOfWeekMonth extends Array<number> {
```
Comment (horiuchi) [inline node-calendar/node-calendar.d.ts]: I propose this base type would use tuple type(e.g. [number, number]) instead of number of array.

Comment (luzianz) [inline node-calendar/node-calendar.d.ts]: @horiuchi Thank you for that. I was wondering how to do that in typescript.

**Comment by luzianz** [comment]
@horiuchi Is this okay?

**Comment by horiuchi** [comment]
@luzianz Sorry for delayed check.

LGTM :+1:

**Comment by luzianz** [comment]
@horiuchi thanks!

**Pull request merged**